src/schema/rules/tabular_data/physio.yaml

```diff
@@ -7,3 +7,31 @@ PhysioColumns:
     respiratory: optional
     trigger: optional
   additional_columns: allowed
+
+PhysioEyeTracking:
+  selectors:
+    - PhysioType == "eyetrack"
+  extension:
+    - '.tsv.gz'
+  columns:
+    et_timestamp: recommended
+    x_coordinate: required
+    y_coordinate: required
+    pupil_size: optional
+  initial_columns:
+    - et_timestamp
+    - x_coordinate
+    - y_coordinate
```

Comment:
@Remi-Gau this is my unsuccessful attempt to have the timestamp be first even though it is recommended. It doesn't render first in the tables.

Reply:
ha, I see.
So yeah very possible that the table rendering macro does not take this into consideration even if the validator does.

Reply:
probably better addressed in a separate PR

Reply:
Okay, so from the schema point of view, this is correct. I agree this can be a separate PR if the only problem is the macro.

Reply:
The converter does place it as first column, I reordered it in the code

Reply:
Should it be named et_timestamp now? Or was it just for the macro? I'm updating the converter right now.

Reply:
This is an internal reference for the spec. The recommended column name is 'timestamp', as it reads in the table of the rendered spec.

Reply:
cc @rwblair for a wrinkle in initial column enforcement.

Reply:
Should be doable. Opened a validator issue for it.
https://github.com/bids-standard/bids-validator/issues/1948